lib/engine-addon.js

```diff
@@ -8,6 +8,7 @@ var writeFile = require('broccoli-file-creator');
 var Babel = require('broccoli-babel-transpiler');
 var concat = require('broccoli-concat');
 var DependencyFunnel = require('broccoli-dependency-funnel');
+var defaults = require('lodash.defaults');
```

Comment:
Can we just use `Object.assign` instead? I'm fairly certain we only need to support Node 4+ at this point due to our Ember-CLI support policy